data/json/monsters/insect_spider.json

```diff
@@ -2294,6 +2294,155 @@
+    "copy-from": "mon_stag_beetle_giant",
+    "proportional": { "hp": 0.5, "morale": 0.5, "melee_dice_sides": 0.67 },
+    "volume": "200 L",
+    "weight": "300 kg",
+    "diff": 1,
+    "melee_cut": 4,
+    "armor_bash": 24,
+    "armor_cut": 32,
+    "armor_bullet": 20,
+    "armor_acid": 8,
+    "armor_fire": 2,
+    "special_attacks": [
+      {
+        "id": "stag_smash",
+        "throw_strength": 40,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 20, "armor_multiplier": 0.5 } ]
+      }
+    ],
+    "upgrades": { "age_grow": 28, "into": "mon_stag_beetle_giant" },
+    "delete": {
+      "reproduction": { "baby_egg": "egg_stag_beetle", "baby_count": 1, "baby_timer": 20 },
+      "flags": [ "DESTROYS", "PUSH_VEH" ]
+    }
+  },
+  {
+    "id": "mon_stag_beetle_giant",
+    "type": "MONSTER",
+    "name": { "str": "massive stag beetle" },
+    "description": "This enormous stag beetle has reached mutated maturity, but continues to grow.  Thankfully, that growth is exposing vulnerable cracks of flesh in the strained chitin.",
+    "default_faction": "stag_beetle",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "400 L",
+    "weight": "600 kg",
+    "diff": 2,
+    "hp": 350,
+    "speed": 80,
+    "material": [ "iflesh" ],
+    "symbol": "S",
+    "color": "brown",
+    "aggression": 40,
+    "morale": 30,
+    "melee_skill": 5,
+    "melee_dice": 6,
+    "melee_dice_sides": 6,
+    "melee_cut": 9,
+    "dodge": 1,
+    "armor_bash": 60,
+    "armor_cut": 72,
+    "armor_bullet": 50,
+    "armor_acid": 18,
+    "armor_fire": 8,
+    "weakpoints": [
+      { "name": "a soft section of chitin", "armor_mult": { "physical": 0.33 }, "coverage": 20 },
+      { "name": "a crack in the chitin", "armor_mult": { "physical": 0 }, "coverage": 5 }
+    ],
+    "mountable_weight_ratio": 6,
+    "vision_day": 14,
+    "vision_night": 12,
+    "harvest": "arachnid",
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "fear_triggers": [ "FIRE" ],
+    "special_attacks": [
+      {
+        "id": "stag_smash",
+        "throw_strength": 60,
+        "hit_dmg_u": "The %1$s rams you in the %2$s!",
+        "hit_dmg_npc": "The %1$s rams <npcname> in the %2$s!",
+        "no_dmg_msg_u": "The %1$s rams your %2$s, but you hold steady!",
+        "no_dmg_msg_npc": "The %1$s rams <npcname>'s %2$s, but they hold steady!",
+        "miss_msg_u": "The %s flicks it's horns at you, but you dodge!",
+        "miss_msg_npc": "The %s flicks it's horns at <npcname>, but they dodge!"
+      }
+    ],
+    "upgrades": { "age_grow": 28, "into": "mon_stag_beetle_mega" },
+    "reproduction": { "baby_egg": "egg_stag_beetle", "baby_count": 1, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN", "WINTER" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "GOODHEARING",
+      "BASHES",
+      "DESTROYS",
+      "PATH_AVOID_FIRE",
+      "PATH_AVOID_FALL",
```

Comment:
```suggestion
      "PATH_AVOID_DANGER_1",
```

Reply:
Thank you, don't know why I blanked on this.